packages/button/src/ButtonBase.ts

```diff
@@ -181,22 +195,26 @@ export class ButtonBase extends ObserveSlotText(LikeAnchor(Focusable), '', [
     }
 
     private manageAnchor(): void {
+        // for a link
```

Comment:
Can we remove the comments from the code before merging?

Reply:
I actually don’t mind these comments... maybe we should align as a team on the balance between self-documenting code and explicit comments.

Reply:
I agree with Ruben. I was pairing with @caseyisonit and the comments we added were because we felt they should be there.

Reply:
i agree on aligning, i am a fan of self-documenting code and explicit comments